README.md

```diff
@@ -80,27 +81,26 @@ We will backport bugfixes--but not new features--into older versions of
 
 #### Compatibility matrix
 
-|                     | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|                     | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 | Kubernetes 1.8 |
 |---------------------|----------------|----------------|----------------|----------------|----------------|
-| client-go 1.4       | +              | ✓              | -              | -              | -              |
-| client-go 1.5       | +              | +              | -              | -              | -              |
-| client-go 2.0       | +              | +              | ✓              | -              | -              |
-| client-go 3.0       | †              | †              | †              | ✓              | -              |
-| client-go 4.0       | †              | †              | †              | +              | ✓              |
-| client-go HEAD      | †              | †              | †              | +              | +              |
+| client-go 1.4       | ✓              | -              | -              | -              | -              |
+| client-go 1.5       | +              | -              | -              | -              | -              |
+| client-go 2.0       | +              | ✓              | -              | -              | -              |
+| client-go 3.0       | -              | -              | ✓              | -              | -              |
+| client-go 4.0       | -              | -              | +              | ✓              | -              |
+| client-go 5.0       | -              | -              | -              | -              | ✓              |
+| client-go HEAD      | -              | -              | -              | -              | +              |
 
 Key:
 
 * `✓` Exactly the same features / API objects in both client-go and the Kubernetes
   version.
-* `+` client-go has features or api objects that may not be present in the
+* `+` client-go has features or API objects that may not be present in the
   Kubernetes cluster, but everything they have in common will work. Please
   note that alpha APIs may vanish or change significantly in a single release.
-* `†` client-go has new features or api objects, and some APIs running in the
-  cluster may have been deprecated and removed from client-go. But everything
-  they share in common (i.e., most APIs) will work.
 * `-` The Kubernetes cluster has features the client-go library can't use
-  (additional API objects, etc).
+  (additional new API, or deprecated API, etc). But everything
+  they share in common (i.e., most APIs) will work.
```

Comment:
I suspect we are missing the case where an older version of the client has an API which is no longer supported by the the newer cluster. This is similar to the '+' case (new client with a cluster so old it does not yet support the feature). However I do not see any such warning for things line client 1.4 with Kubernetes 1.8, so I suspect we are missing this case.

Reply:
Comments addressed. PTAL. Thank you.